generated/nidigital/session.py

```diff
@@ -664,11 +664,9 @@ def apply_tdr_offsets(self, num_offsets, offsets):
         You can specify a subset of repeated capabilities using the Python index notation on an
         nidigital.Session repeated capabilities container, and calling this method on the result.:
 
-            session.channels[0,1].apply_tdr_offsets(num_offsets, offsets)
+            session.channels[0,1].apply_tdr_offsets(offsets)
```

Comment:
I'm pretty confused why this is only a documentation change.
If you look at the old code to teh left, on line 677:
```
        num_offsets_ctype = _visatype.ViInt32(0 if offsets is None else len(offsets))  # case S160		
```
is the same as the right hand side.

Why was parameter `num_offsets` not being used?
Was the generator generating valid yet "incorrect" code?
Or more likely, should the generator have caught a mistake in metadata and broken at codegen time?

@texasaggie97

Reply:
```
{
    'direction': 'in',
    'name': 'offsets',
    'size': {
        'mechanism': 'passed-in',
        'value': 'numOffsets'
    },
    'type': 'ViReal64[]'
}
```

This is how metadata looked like before my fix, which was to change `mechanism` to `len`. Generated code used passed in list to figure out the size that needs to be passed to C DLL, but also added `numOffsets` as API method parameter.

Reply:
The public method has a parameter `num_offsets`.
Metadata says `'size': {'mechanism': 'passed-in', 'value': 'numOffsets'}`
But the generated code never uses `numOffsets`. In line 677 (left), variable `num_offsets_ctype` is initialized with the length of `offsets`.